package.json

```diff
@@ -72,6 +72,9 @@
     "jest": "^27.5.1",
     "node-plop": "^0.26.2",
     "plop": "^2.7.4",
+    "postcss": "8.4.4",
```

Comment:
why we need now postcss as dev dependency?

Reply:
I haven't added it, it was just moved
But it's being used e.g. here 
<img width="701" alt="image" src="https://github.com/mondaycom/monday-ui-style/assets/104433616/d642ede4-3286-4823-9eb9-ecf1e57214fa">